ONSdigital/eq-questionnaire-runner
# Add uwsgi run option

### What is the context of this PR?
Adds the option to run under uWSGI. There are three options within this - plain uWSGI, threaded or asynchronous (via gevent).

This branch has been used to run the tests in https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/55.

### How to review 
- Run the app locally with the Makefile commands `run-uwsgi`, `run-uwsgi-threads` or `run-uwsgi-async`
- Check that `run-gunicorn` works as it did previously
- Run under Docker
